### PR TITLE
Feature/up ncp limit

### PR DESCRIPTION
**Pull request by serby**
Speeding up NCP

### CODE REVIEW

**Comment by serby** [comment]
@microadam Please QA and let me know, when or how this needs to be deployed.

File: .jscsrc [hunk 1]
```diff
@@ -1,13 +1,15 @@
 { "excludeFiles":
   [ "node_modules/**"
   , "coverage"
+  , "site/themes/*/assets/js"
```
Comment (microadam) [inline .jscsrc]: @serby could we remove these ignores please? not really applicable here

**Comment by microadam** [comment]
@serby Thanks a lot for this!

checked out the branch and there are a few lint errors. Assuming your editor didn't pick them up? Tests pass though and code looks fine otherwise.

I will leave you to fix the lint errors so you can make sure your editor is definitely configured correctly. 

In terms of deployment, will need to update the ansible role with the new version of this package (once published) and then run it on whatever platform you wish to have it on for immediate testing

**Comment by serby** [comment]
Good spot. Done and done.

**Comment by serby** [comment]
ShortList, RIBA and Localworld

**Comment by microadam** [comment]
Thanks!

**Pull request merged**